lib/HelloSignResource.js

```diff
@@ -188,12 +188,14 @@ HelloSignResource.prototype = {
             } else if ('content-transfer-encoding' in resHeaders && resHeaders['content-transfer-encoding'] == "binary"){
                 callback.call(self, null, res);
             // Simplified response for code-only statuses
-            } else if ('content-length' in resHeaders && resHeaders['content-length'] == "1") {
+            } else if ('content-length' in resHeaders && resHeaders['content-length'] <= "1") {
```

Comment:
I am unsure of the previous call to have this as ==1 but I have confirmed with testing and review of the documentation that a status code of 200 will result in a Content-Type of 0. So checking for a Content-length of less than or equal to 1 will suffice in this case. This is based on initial review, still need to test functionality.

Reply:
@latoyazamill do you know why we're comparing two strings here? It feels a little bit flimsy to type-convert both operands. It might be better to do something like

```
parseInt(resHeaders['content-length`], 10) <= 1
```

Where at least in this case we're making a _conscious_ effort to convert (`parseInt`) one side of the operator to the other operand of the same type

Reply:
I cannot speak to this @nathanbuchar. That was before my time with the SDK. Did you want this concern to be refactored for this PR? If so, @jpescador will need to do so in order for this to be merged. If not, we can merge this PR as is (after your final review) and then we can refactor internally. Leaving this up to your best judgement. Thanks!

Reply:
I'm sure it's fine as-is, just wanted to note it